content/attacks/posts/2024-02-23-BitForex.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2024-02-23
+target-entities: BitForex
+entity-types:
+  - Exchange
+  - Custodian
+attack-types:
+  - Exit Scam
+  - Wallet Hack
+title: "BitForex's Suspicious Outflows: Hack or Exit Scam?"
+loss: 56500000
+---
+
+## Summary
+
+Since February 23, 2024, BitForex, a crypto exchange operational since 2017, ceased processing withdrawals amidst unexplained outflows of about $56.5M worth of crypto from its hot wallets. The absence of communication from BitForex, coupled with the recent departure of its CEO Jason Luo, has raised concerns over a potential inside job or exit scam. 
+
+## Attackers
+
+The identity of the attacker is unknown. The following addresses are associated with this attack:
+
+- Ethereum Wallet:
+  - [0xdcacd7eb6692b816b6957f8898c1c4b63d1fc01f](https://etherscan.io/address/0xdcacd7eb6692b816b6957f8898c1c4b63d1fc01f)
+    
+- Tron Wallet:
+  - [TQcnqaU4NDTR86eA4FZneeKfJMiQi7i76o](https://tronscan.org/#/address/TQcnqaU4NDTR86eA4FZneeKfJMiQi7i76o)
+   
+- Bitcoin Wallet:
+  - [3DbbF7yxCR7ni94ANrRkfV12rJoxrmo1o2](https://www.blockchain.com/explorer/addresses/btc/3DbbF7yxCR7ni94ANrRkfV12rJoxrmo1o2)
+
+## Losses
+
+The total loss from the BitForex hack amounted to approximately [$56.5 million](https://twitter.com/zachxbt/status/1762028433574650347), distributed across the following networks:
```

Comment:
There is no evidence that it was a hack. It's more likely to be an exitscam. [This source](https://bits.media/gonkongskiy-regulyator-obvinil-kriptobirzhu-bitforex-v-moshennichestve/) might be useful.